tests/constants.ts

```diff
@@ -38,7 +37,7 @@ export const CLUSTER = 'devnet';
 
 // ----------------------------------------------------------------------------
 export const uxdProgramId: PublicKey = new PublicKey(
-  jsonIdl['metadata']['address']
+  'EqMrasrCXsbE6mpXaopUH1qip7auwX4AF5UWWpXaLr6u'
```

Comment:
need to take extra care on checking whether this program id matched the one on `./target/deploy/uxd-keypair.json`, but let's go for this changes coz the removal of program id with `anchor build` is indeed super annoying

Reply:
For sure, True!

Before we had 3 places to double check: lib.rs, uxd.json, Anchor.toml

Now we have 3 places to double check: lib.rs, constants.ts, Anchor.toml

No real improvements here, just avoiding the auto-generation problem that nukes uxd.json